src/main/resources/cftlib-am-role-assignments.json

```diff
@@ -1,136 +1,136 @@
 [
   {
-    "email": "TEST_SYSTEM_USER@mailinator.com",
-    "id": "93b108b7-4b26-41bf-ae8f-6e356efb11b3",
+    "email": "TEST_CASE_WORKER_USER@mailinator.com",
+    "id": "93b108b7-4b26-41bf-ae8f-6e356efb11b1",
```

Comment:
This is needs to match the user id that the idam simulator generates - UUID.nameUUIDFromBytes(request.getEmail().getBytes()).toString();